govc/events/command.go

```diff
@@ -54,6 +59,28 @@ func (cmd *events) Process(ctx context.Context) error {
 	return nil
 }
 
+func (cmd *events) printEvents(ctx context.Context, page []types.BaseEvent, m *event.Manager) error {
```

Comment:
Might be useful to move this to something like:
```go
func (m *event.Manager) Write(ctx context.Context, page []types.BaseEvent, w io.Writer) error
```
Then the same helper can be used outside of govc to write events to any io.Writer (Stdout, File, etc).